### ISSUE TITLE
Implement "About: Mission and History" page

### ISSUE THREAD

**Issue by noahmanger** [18F/fec-cms#700]
So that users can understand the mission and history of the FEC, implement a page under "About the FEC" that contains this information.

---

🖌  [Design](https://cloud.githubusercontent.com/assets/11636908/21554663/7aef6e6e-cddf-11e6-81fa-33369427572e.png) | 📜  [Content](https://docs.google.com/document/d/10GC39hUAzucOngCZ-9dSZ_LmG0P1mnUj4zPL9gjmK1I/edit) | 🗣  [Discussion](https://github.com/18F/fec-cms/issues/673) 

This page can build off of the new `ResourcePage` model, but will require adding a couple new modules. Once those modules are added we'll need to wait until this template is deployed in the next release and then we can make the new page and add the content.

**Completion criteria:**
- [x] Export video thumbnail icon - @jenniferthibault
- [x] Implement module for the video thumbnail
- [x] Implement way to set the color of the breadcrumbs bar
- [x] Implement module for action button ("View timeline") at the bottom

**Comment by noahmanger** [comment]
Ok, this was a pretty simple extension of the new ResourcePage template. [Here's how it's looking](https://cloud.githubusercontent.com/assets/1696495/21663559/cdd3cef8-d295-11e6-9ab6-2fcf16a75eb7.png).

One place where I diverged slightly from the designs is in the way I did these links:
![image](https://cloud.githubusercontent.com/assets/1696495/21663570/ed687f48-d295-11e6-9958-9ff5c92662ca.png)

I just used soft returns (thanks Jen!) to separate the links from the text above and made bold. This is a lot simpler than trying to figure out a new module for intermixing list items and links, but I'm open to revising it.
